x/model/handler_test.go

```diff
@@ -698,6 +698,9 @@ func TestHandler_UpdateModelVersion(t *testing.T) {
 
 	// try update not present model version
 	msgUpdateModelVersion := NewMsgUpdateModelVersion(setup.Vendor)
+	msgUpdateModelVersion.OtaUrl = ""
```

Comment:
Why do we need to set these values to default in so many places? Aren't they default by default? If not, then it cane sent to default in `NewMsgUpdateModelVersion` method (in one place).

Reply:
Fixed